.abs/main.yaml

```diff
@@ -0,0 +1,4 @@
+replace-chart-version-with-git: true
+replace-app-version-with-git: true
+chart-dir: ./helm/athena
```

Comment:
I am thinking this should be `./helm/happa`, no?

Reply:
Yes, missed that when copy-pasting. Thank you.
Weird that I didn't notice it in any other way.

Reply:
I guess in the `push-to-app-catalog` job it is overwritten.

If we continue to maintain two charts in this repository, it might be best to remove the `chart-dir` from the `.abs.` file.